css-positioning/fixed.css

```diff
@@ -0,0 +1,11 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+  position: fixed;
+  top: 330px;
+  left: 450px;
```

Comment:
If you make the title `width: 100%`, this won't be necessary, since `text-align: center;` has already been applied